src/main/java/com/gw/web/GeoweaverController.java

```diff
@@ -417,50 +422,50 @@ public String showResetPasswordForm(@Param(value = "token") String token, Model
 	 */
 	@RequestMapping(value = "/log", method = RequestMethod.POST)
     public @ResponseBody String one_history(ModelMap model, WebRequest request){
-		
+
 		String resp = null;
-		
+
 		try {
-			
+
 			String type = request.getParameter("type");
-			
+
 			String hid = request.getParameter("id");
-			
+
 			if(type.equals("process")) {
-				
+
 				resp = pt.one_history(hid);
-				
+
 			}else if(type.equals("workflow")) {
-				
+
 				resp = wt.one_history(hid);
-				
+
 			}else if(type.equals("host")) {
-				
+
 				resp = ht.one_history(hid);
-				
+
 			}
-			
+
 		}catch(Exception e) {
-			
+
 			e.printStackTrace();
-			
+
 			throw new RuntimeException("failed " + e.getLocalizedMessage());
-			
+
 		}
-		
+
 		return resp;
-		
+
 	}
 
 	@RequestMapping(value= "/skip_workflow_process", method = RequestMethod.POST)
 	public @ResponseBody String skip_process(ModelMap model, WebRequest request){
 
 		String resp = null;
-		
+
 		try {
-			
+
 			String workflow_id = request.getParameter("workflowid");
-			
```

Comment:
There are so many space changes. Can you do these changes in a separate PR? It is very confusing and not easy to find the changes that really matter.